src/function/9pfs.c

```diff
@@ -0,0 +1,103 @@
+/*
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ */
```

Comment:
Please include the copyright statement before the license grant.

Reply:
Those two files are actually copied from somewhere in the project and reused. This means that we have to check other functions as well. I will add it here and create a issue for the other files.

Reply:
Fixed